src/main/java/org/spongepowered/mod/mixin/core/event/player/MixinEventPlayerInteractBlock.java

```diff
@@ -79,4 +85,15 @@ public EntityInteractionType getInteractionType() {
         return Optional.fromNullable(new Cause(null, this.entityPlayer, null));
     }
 
+    @Override
+    public Optional<Vector3d> getClickedPosition() {
+        return Optional.of(VecHelper.toVector(this.pos).toDouble());
```

Comment:
But than this mixin is kinda useless. As the event will have to be called from ```NetHandlerPlayServer.processPlayerBlockPlacement```. 

Isn't it possible to figure out the clicked position from the players position?